src/main/java/com/minecolonies/core/client/gui/modules/WindowListRecipes.java

```diff
@@ -184,12 +184,18 @@ public void updateElement(final int index, @NotNull final Pane rowPane)
                 List<ItemStack> displayStacks = recipe.getRecipeType().getOutputDisplayStacks();
                 icon.setItem(displayStacks.get((lifeCount / LIFE_COUNT_DIVIDER) % (displayStacks.size())));
 
-                if (!module.isRecipeAlterationAllowed())
+                final Button removeButton = rowPane.findPaneOfTypeByID(BUTTON_REMOVE, Button.class);
+                if (removeButton != null)
                 {
-                    final Button removeButton = rowPane.findPaneOfTypeByID(BUTTON_REMOVE, Button.class);
-                    if (removeButton != null)
+                    if (module.isRecipeAlterationAllowed())
+                    {
+                        removeButton.setVisible(true);
+                        removeButton.setEnabled(recipe.getRecipeSource() == null || Screen.hasControlDown());
```

Comment:
I'm not sure locking it behind holding CTRL is a good idea without some kind of targeted information that it works this way. People are just going to see a previously unlocked button now disabled without any kind of information on how it actually works now.

We should come up with a proper idea and not shoehorn a quick and dirty fix into it.